coderd/httpmw/apikey_test.go

```diff
@@ -991,3 +991,237 @@ func TestAPIKey(t *testing.T) {
 		require.Equal(t, http.StatusOK, res.StatusCode)
 	})
 }
+
+func TestAPIKeyExpiryRefresh(t *testing.T) {
+	t.Parallel()
+
+	t.Run("ShortLivedKeyRefresh", func(t *testing.T) {
+		t.Parallel()
+		var (
+			db, _ = dbtestutil.NewDB(t)
+			user  = dbgen.User(t, db, database.User{})
+			now   = time.Now()
+			// 2 minute lifetime
+			lifetime = int64(120)
+			// Key expires in 30 seconds (less than half the 2-minute lifetime)
+			sentAPIKey, token = dbgen.APIKey(t, db, database.APIKey{
```

Comment:
All the other tests use `client.CreateToken`, could we not do that and pass in the desired lifetime?